libraries/AP_GPS/AP_GPS.h

```diff
@@ -313,6 +326,14 @@ class AP_GPS
         return vertical_accuracy(primary_instance, vacc);
     }
 
+    // Time accuracy in seconds
+    bool time_accuracy(uint8_t instance, float &t_acc) const;
```

Comment:
Why is there no implementation here?

Reply:
Sorry. My bad. I was looking at the height_above_ellipsoid(uint8_t instance) function above and started wondering why the same pattern weren't repeated here.